src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -5808,6 +6026,20 @@ components:
       description: DUOS dataset identifier
       example: DUOS-123456
 
+    Tag:
+      type: string
```

Comment:
I was also thinking a bit about validating the tag value. It looks like it currently allows spaces, characters, etc. Would it make sense from a security perspective to narrow down what is allowed? I could see allowing just letters, dashes and underscores. But, also worth considering having feature parity with Terra. 

One nuance: If you try to add a comma to the tag, it is added as two tags. This seems rather harmless.

Reply:
Thanks for this feedback!  Closing the loop after our discussion after stand-up today:

- Max length - since these tags aren't restricted by limitations imposed by external systems (ex. BigQuery) and are only stored in our Postgres DB, we've elected not to add a max length.  We'll want to think about how to render very long tags in TDR UI once we get to that work.
- Validating tag value - we're preferring to keep feature parity with Terra which generally allows all characters (except fixing an order of operations bug where we want to strip whitespace from inputted tags, then deduplicate them).  We tried to "hack.edu" our way into a bad tag value in Terra UI and everything seemed to behave well.
- Commas in tags - this was caused by tags to add / remove being set as query parameters.  I changed these endpoints to use response bodies instead and we can now add / remove tags with commas.